var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -271,26 +288,64 @@ def bootstrap_args(self):
 
         # When building our own private copy of curl then we need to properly
         # enable / disable oepnssl
-        if '+ownlibs' in spec:
+        if '+openssl' in spec and '+ownlibs' not in spec:
```

Comment:
Did you read the comment one line above? `cmake +ownlibs` bootstraps curl but does not bootstrap openssl (presumably because of security issues / system certificates), so this change can't be correct.